### DISCUSSION TITLE
println!("Let's talk about logging")

### DISCUSSION THREAD

**Discussion by bobmcwhirter** [guacsec/trustify#802] [General]
Let's take a moment and think about logging. I understand the irony of this, given I'm Mr. `println!`.

I think certainly INFO/WARN/ERROR have their place, but we should use them to express information relevant to *operators* of the system. The admin who might check logs or dig around in splunk.

For things like "this SBOM is bad", we need to surface that to the UI somehow, as the SRE operating the system probably won't care.

For our purposes, DEBUG/TRACE can exist for "this SBOM is bad" sorta thing, generally suppressed.

Yeah? No?

**Comment by jcrossley3** [comment]
> I understand the irony of this, given I'm Mr. `println!`.

:rofl:

**Comment by helio-frota** [comment]
yes :+1:

**Comment by ctron** [comment]
I agree and I think this relates to https://github.com/trustification/trustify/discussions/756
